### PR TITLE
add issue&PR template , update .gitignore

### LINKED ISSUE #7: set .gitignore
Please write any language or framework to add on .gitignore
@psh2849 @seoh02h @Jsueeee

### LINKED ISSUE #8: Make issue template

### LINKED ISSUE #9: Make PR template

### PR DESCRIPTION
**Pull request by sunnnl**
add ISSUE_TEMPLATE.md and PULL_REQUEST_TEMPLATE.md
As we discussed, I just added basic contents in templates as I like.
Please, give some more ideas :)
Also, update .gitignore with the help of https://www.toptal.com/developers/gitignore !

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Title
+
+## Detials
```
Comment (Jsueeee) [inline .github/PULL_REQUEST_TEMPLATE.md]: Typo Correction!
```suggestion
## Details
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+### as-is
+
+### to-be
+
```
Comment (Jsueeee) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: as-is / to-be idea is perfect! 😁

**Comment by psh2849** [comment]
How about add the issue number in PR template? 
then, we know what issues have you solved !

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+## Expected Behavior
+
+## Current Behavior
+
+## Failure Logs
```
Comment (seohy02) [inline .github/ISSUE_TEMPLATE.md:5]: Great !!

**Review by seohy02 — APPROVED** [review]

**Comment by seohy02** [comment]
> How about add the issue number in PR template?
> then, we know what issues have you solved !

It sounds good !

**Comment by Jsueeee** [comment]
> How about add the issue number in PR template?
> then, we know what issues have you solved !

I'll do it!

**Pull request merged**